homeworks/Homework6/Pets/Parrot.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "Pet.h"
+
+class Parrot : public Pet {
+	int words; // how many words know
```

Comment:
Than you can add method talk and perrot say all words he know) Think how it can be implemented